cpp/testdata/BUILD

```diff
@@ -9,26 +9,26 @@ jsonnet_to_json_test(
     name = "example_test",
     src = "example.jsonnet",
     golden = "example_golden.json",
-    jsonnet = "//:jsonnet",
-    std = "//:std",
+    jsonnet = "//cmd:jsonnet",
+    std = "//stdlib",
 )
 
 jsonnet_to_json_test(
     name = "importing_test",
     src = "importing.jsonnet",
     golden = "importing_golden.json",
     imports = ["."],
-    jsonnet = "//:jsonnet",
-    std = "//:std",
+    jsonnet = "//cmd:jsonnet",
+    std = "//stdlib",
 )
 
 jsonnet_to_json_test(
     name = "invalid_test",
     src = "invalid.jsonnet",
     golden = "invalid.out",
     error = 1,
-    jsonnet = "//:jsonnet",
-    std = "//:std",
+    jsonnet = "//cmd:jsonnet",
+    std = "//stdlib",
```

Comment:
You may be right that it's unnecessary, but the jsonnet_to_json_test rule defined in the bazel repo has a default value for std, which is simply wrong for these targets.